### ISSUE TITLE
UBS daily low price data error on 2022-08-30 (open 16.12, high 16.14, low 0.9242, close 15.95)

### ISSUE THREAD

**Issue by NeilJustice** [polygon-io/issues#267]
**URL**
https://api.polygon.io/v2/aggs/ticker/UBS/range/1/day/2022-08-30/2022-08-30?adjusted=false&apiKey=

Polygon daily data for UBS on 2022-08-30:

Open: 16.12
High: 16.14
**Low: 0.9242 (!)**
Close: 15.95

<img width="1213" alt="UBSLow" src="https://github.com/polygon-io/issues/assets/1912242/b9210a29-3d45-4d21-8add-336c92b5faab">

Although there is no one source of open-high-low-close truth, with TradingView being the most referenced data provider in the world, TradingView's data values for the same symbol and day are presented here for reference:

![image](https://github.com/polygon-io/issues/assets/1912242/a1f34ba9-2b14-47cc-9a0b-c686a18b933e)

**Comment by AHangstefer** [comment]
Hi! Thanks for reaching out. I checked the trades for UBS on 2022-08-30 to make sure the 0.9242 price trade exists. 
This is the query I made:

https://api.polygon.io/v3/trades/UBS?order=desc&timestamp=2022-08-30&limit=50000&apiKey=**

It looks like that value is coming from a dark pool trade which we include in our Aggregates:

conditions: [3]
exchange: 4
id: "71697119463245"
participant_timestamp: 1661870314597000000
price: 0.9242
sequence_number: 2654517
sip_timestamp: 1661893504974560300
size: 100000
tape: 1
trf_id: 12
trf_timestamp: 1661893504974188000
